Extension/package.nls.json

```diff
@@ -17,67 +17,96 @@
+    "c_cpp.configuration.vcFormat.indent.preprocessor.oneLeft.description": "to the left of the current code indentation, by the amount specified in the Editor: Tab Size setting",
+    "c_cpp.configuration.vcFormat.indent.preprocessor.leftmostColumn.description": "at the leftmost edge of the code",
+    "c_cpp.configuration.vcFormat.indent.preprocessor.none.description": "preprocessor directives will not be formatted",
+    "c_cpp.configuration.vcFormat.indent.accessSpecifiers.description": "Access specifiers are indented relative to class or struct definitions by the amount specified in the Editor: Tab Size setting",
     "c_cpp.configuration.vcFormat.indent.namespaceContents.description": "Code is indented relative to its enclosing namespace by the amount specified in the Editor: Tab Size setting",
     "c_cpp.configuration.vcFormat.indent.preserveComment.description": "Indentation of comments is not changed during formatting operations",
-    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.namespace.description": "Opening braces are moved to a new line, kept on the same line and a space is added before each or leaves open braces where entered",
-    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.type.description": "Opening braces are moved to a new line, kept on the same line and a space is added before each or leaves open braces where entered",
-    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.function.description": "Opening braces are moved to a new line, kept on the same line and a space is added before each or leaves open braces where entered",
-    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.block.description": "Opening braces are moved to a new line, kept on the same line and a space is added before each or leaves open braces where entered",
-    "c_cpp.configuration.vcFormat.newLine.scopeBracesOnSeparateLines.description": "Moves opening and closing braces on separate lines",
-    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.lambda.description": "Opening braces are moved to a new line, kept on the same line and a space is added before each or leaves open braces where entered",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.namespace.description": "The position of opening braces for namespaces",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.type.description": "The position of opening braces for type definitions",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.lambda.description": "The position of opening braces for lambda functions",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.function.description": "The position of opening braces for functions",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.block.description": "The position of opening braces for control blocks",
+    "c_cpp.configuration.vcFormat.newLine.beforeOpenBrace.newLine.description": "opening braces are moved to a new line",
```

Comment:
Are these descriptions supposed to all start with a capital letter?

Reply:
The enum values start with lowercase letters